### ISSUE TITLE
XY Plot - Need need to show appropriate information after selecting 'Produce a WMF file for the graph' button

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#3311]
Currently when a user press 'Produce a WMF file for the graph' button on XY plot, there is nothing shown.  A user can't know whether his action is successfully completed or not unless he opens the directory where this file reside. 
### Steps to Reproduce:
1. Open sample.dpf file
2. Select the two tracks files > Right-click > View XY Plot
3. Click on 'Produce a WMF file for the graph' button
### Expected Result
If user's actions is successfully completed, we need to tell the folder and file name of this WMF file is produced.
If the user's action is failed, we need to tell , the file is not produced, and better to tell the reason as well

**Comment by IanMayo** [comment]
Yes, this is useful feedback.  We should indicate success (image has been copied to file) or failure (explain nature of failure).
